kroxylicious-systemtests/src/main/java/io/kroxylicious/systemtests/utils/DeploymentUtils.java

```diff
@@ -71,13 +72,14 @@ private DeploymentUtils() {
      * @param namespaceName the namespace name
      * @param deploymentName the deployment name
      */
-    public static void waitForDeploymentReady(String namespaceName, String deploymentName) {
+    public static boolean waitForDeploymentReady(String namespaceName, String deploymentName) {
```

Comment:
since the method only ever returns true (or thows an exception), what's the point of the return value